web/auth/jwt_test.go

```diff
@@ -1,6 +1,67 @@
 package auth_test
 
-import "testing"
+import (
+	"testing"
 
-func TestUpdateJWT(t *testing.T) {
+	"github.com/autograde/quickfeed/ag"
```

Comment:
Imported twice causing the linter to complain. For now I prefer to use `pb` since that's what we use in other places.

In the future, I think I want to move the `ag` package to a new `proto` package, which seems to be a common place for proto files... but not sure... I would require a pretty big PR that touches many packages...

Reply:
Yeah, double importing was an accident. Fixed.